src/App.tsx

```diff
@@ -10,32 +11,28 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+  const selectedTab : Tab = tabs.find(
+    tab => tab.id === selectedTabId,
+  ) || tabs[0];
+
+  const { title, content } = selectedTab;
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        `Selected tab is
+        {' '}
+        {title}
+        `
```

Comment:
```suggestion
        Selected tab is
        {' '}
        {title}
```

Reply:
![image](https://user-images.githubusercontent.com/98764641/227889792-aa9f71a9-c6d6-4336-a69f-548608ec05ae.png)

Reply:
```suggestion
        {`Selected tab is {title}`}
```

Reply:
or this way